.github/workflows/ci-tk.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         version: [3.11]
-        os: [ubuntu-22.04, nodai-amdgpu-mi300-x86-64, nodai-amdgpu-mi250-x86-64]
+        os: [ubuntu-22.04, linux-mi300-1gpu-ossci-iree-org, nodai-amdgpu-mi250-x86-64]
```

Comment:
The name "os" is still a poor fit here as mentioned in my prior comment, but this is okay for an incremental change. Could also refactor _prior_ to this PR and then have this PR just switch the label and have the conditionals remain unchanged.

Reply:
Ok.  matrix.os is used throughout a lot workflows though

Reply:
Yep. We can clean that up in a separate PR.